CHANGELOG.md

```diff
@@ -15,6 +15,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 - The `--help` output was reworked with a new colorful syntax
 
+### Fixed
+
+- OSC 104 not reseting all colors when used without parameters
```

Comment:
Could we move this change to a separate PR? The two changes are completely unrelated.

Reply:
I put it here because they're both from the same GH issue but I can make a separate PR for it.

Reply:
Github issue doesn't matter, just open a new PR for every separate feature please.

Reply:
Took me a while to get git to do what I wanted but the OSC 104 related stuff from this branch and will open another PR for that after this one